app-misc/anki/anki-2.1.15.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+PYTHON_REQ_USE="sqlite"
+
+inherit desktop python-single-r1 xdg
+
+DESCRIPTION="A spaced-repetition memory training program (flash cards)"
+HOMEPAGE="https://apps.ankiweb.net"
+SRC_URI="https://apps.ankiweb.net/downloads/current/${P}-source.tgz -> ${P}.tgz"
+
+S="${WORKDIR}/${P}"
+
+LICENSE="AGPL-3+ BSD MIT GPL-3+ CC-BY-SA-3.0 Apache-2.0 CC-BY-2.5"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="latex +recording +sound test"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}
+	$(python_gen_cond_dep '
+		>=dev-python/PyQt5-5.12[gui,svg,widgets,${PYTHON_MULTI_USEDEP}]
```

Comment:
I think you can convert these to [PYTHON_USEDEP](https://devmanual.gentoo.org/eclass-reference/python-single-r1.eclass/index.html).

Reply:
Like I mentioned before I'm new at this and don't have a lot of context .  Reading <https://dev.gentoo.org/~mgorny/python-guide/migration.html?highlight=python_usedep> has left be unsure what the preferred form is at this time and I'm unsure what problem using one over the other would solve for this ebuild.

Reply:
Just style:
```During a transitional period, PYTHON_USEDEP was banned entirely and PYTHON_MULTI_USEDEP was used instead. Today both are permitted and the former is recommended.```